pkgs/tools/filesystems/xfsdump/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchgit
+, attr
+, gettext
+, autoconf
+, automake
+, ncurses
+, libtool
+, libuuid
+, libxfs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfsdump";
+  version = "3.1.10";
+
+  src = fetchgit {
+    url = "git://git.kernel.org/pub/scm/fs/xfs/xfsdump-dev.git";
+    rev = "3607469d0a8ca3513bfb6093a0da9932dddc8979";
+    sha256 = "sha256-rvclxQUJ80J0+RuwjyK3LyTvSZnDmzvmgboAPFsXFYM=";
+  };
+
+  nativeBuildInputs = [
+    autoconf automake gettext libtool 
```

Comment:
Can't this use `autoreconfHook` instead?
Or build from a release tarball?

Reply:
I have switched to a tarball, but using `autoreconfHook` gives this linking error that I can't quite figure out:
```
/build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:3200: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_scsitape.o: in function `do_seek_mark':
/build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:1279: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_scsitape.o: in function `do_next_mark':
/build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:1606: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:1526: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:1645: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_scsitape.o:/build/xfsdump-dev-3.1.10/dump/drive_scsitape.c:1578: more undefined references to `_' follow
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_simple.o: in function `do_get_write_buf':
/build/xfsdump-dev-3.1.10/dump/drive_simple.c:1205: undefined reference to `min'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_simple.o: in function `do_read':
/build/xfsdump-dev-3.1.10/dump/drive_simple.c:660: undefined reference to `min'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_simple.o: in function `ds_instantiate':
/build/xfsdump-dev-3.1.10/dump/drive_simple.c:371: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_simple.c:286: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_simple.c:330: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_simple.o: in function `do_begin_write':
/build/xfsdump-dev-3.1.10/dump/drive_simple.c:969: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_simple.c:919: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_simple.o:/build/xfsdump-dev-3.1.10/dump/drive_simple.c:519: more undefined references to `_' follow
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o: in function `do_get_write_buf':
/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1816: undefined reference to `min'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o: in function `ds_instantiate':
/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:549: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:561: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:524: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:515: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:554: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o:/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:615: more undefined references to `_' follow
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o: in function `do_read':
/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:854: undefined reference to `min'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o: in function `do_seek_mark':
/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1166: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o: in function `do_next_mark':
/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1462: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1411: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1502: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1435: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: drive_minrmt.o:/build/xfsdump-dev-3.1.10/dump/drive_minrmt.c:1488: more undefined references to `_' follow
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: content.o: in function `dump_file_reg':
/build/xfsdump-dev-3.1.10/dump/content.c:4023: undefined reference to `max'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: content.o: in function `dump_extent_group':
/build/xfsdump-dev-3.1.10/dump/content.c:4406: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/content.c:4855: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: content.o: in function `dump_file_reg':
/build/xfsdump-dev-3.1.10/dump/content.c:4072: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/content.c:4122: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: content.o: in function `dump_extent_group':
/build/xfsdump-dev-3.1.10/dump/content.c:4475: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: content.o:/build/xfsdump-dev-3.1.10/dump/content.c:4490: more undefined references to `_' follow
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: inomap.o: in function `cb_add':
/build/xfsdump-dev-3.1.10/dump/inomap.c:512: undefined reference to `max'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: inomap.o: in function `inomap_build':
/build/xfsdump-dev-3.1.10/dump/inomap.c:247: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/inomap.c:300: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/inomap.c:333: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/inomap.c:378: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/xfsdump-dev-3.1.10/dump/inomap.c:387: undefined reference to `_'
/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: inomap.o:/build/xfsdump-dev-3.1.10/dump/inomap.c:378: more undefined references to `_' follow
collect2: error: ld returned 1 exit status
```